babel.config.js

```diff
@@ -0,0 +1,48 @@
+
+const defaultPresets = [
+    '@babel/preset-react',
+    [
+        '@babel/preset-env',
+        {
+            modules: 'commonjs'
+        }
+    ]
+];
+
+const defaultPlugins = [
+    ['@babel/plugin-proposal-class-properties', { loose: true }],
+    ['@babel/plugin-proposal-object-rest-spread', { loose: true }],
+    '@babel/plugin-transform-object-assign'
+];
+
+const productionPlugins = [
+    [
+        'transform-react-remove-prop-types',
+        {
+            mode: 'unsafe-wrap'
+        }
+    ],
+    [
+        '@babel/plugin-transform-react-inline-elements'
+    ]
+];
+
+module.exports = {
+    presets: defaultPresets,
+    plugins: defaultPlugins,
+    env: {
+        production: {
+            presets: [
+                [
+                    '@babel/preset-env',
+                    {
+                        targets: {
+                            ie: '11'
+                        }
+                    }
```

Comment:
We should probably be using browserlist to set `@babel/preset-env`. https://github.com/SAP/fundamental-react/blob/master/package.json#L130 or maybe removal it all together if its not used anywhere but for babel. This was inherited from Create React App.